packages/valory/connections/ledger/tests/test_ledger_api.py

```diff
@@ -98,6 +100,16 @@
     ],
 )
 
+fee_history_mock = patch(
+    "web3.eth.Eth.fee_history",
+    return_value=FeeHistory(
+        baseFeePerGas=[Wei(0)],
+        gasUsedRatio=[0],
+        oldestBlock=BlockNumber(0),
+        reward=[[Wei(0)]],
```

Comment:
perhaps a good test would be to also include non zero values here, and also empty values.
The empty values case might be worth doing in cases when using virtual nets, or locally deployed nodes. Highly unlikely to cause issues otherwise i think.

Reply:
a1c91a6320ef8e0c425c7f910624f8ac5871f0d4